tsconfig.json

```
@@ -0,0 +1,65 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    // "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
+    // "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */,
+    "lib": ["ESNext", "DOM"] /* Specify library files to be included in the compilation. */,
```

Since the `package.json` declares compatibility with Node.js 8 I would recommend setting `ES2017` instead of `ESNext`, as to not accidentally use any API not available in Node.js 8.

ref: https://stackoverflow.com/a/50466512/148072

```suggestion
    "lib": ["ES2017", "DOM"] /* Specify library files to be included in the compilation. */,
```